.github/workflows/pr.yaml

```
@@ -21,6 +22,26 @@ jobs:
       - wheel-tests
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.12
+    if: always()
+    with:
+      needs: ${{ toJSON(needs) }}
+  changed-files:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@branch-24.12
+    with:
+      files_yaml: |
+        test_notebooks:
+          - '**'
+          - '!.pre-commit-config.yaml'
+          - '!CONTRIBUTING.md'
```

```suggestion
          - '!CONTRIBUTING.md'
          - '!EXTERNAL_TESTS.md'
```

Most projects have how to run the tests included in a `CONTRIBUTING.md`, but here some testing details are also captured in another file `EXTERNAL_TESTS.md`. PRs that only change that file don't need a re-run of the tests.